src/Language/Haskell/LSP/Control.hs

```diff
@@ -68,41 +68,44 @@ runWithHandles hin hout dp h o recInFp recOutFp = do
   hSetEncoding  hout utf8
 
   -- Delete existing recordings if they exist
-  forM_ [recInFp, recOutFp] $ maybe (return ()) $ \f -> do
-    exists <- doesFileExist f
-    when exists $ removeFile f
+  maybe (return ()) removeFileIfExists captureFp
```

Comment:
perhaps capture against a filename with a timestamp in it.  That way we do not lose something accidentally.

Eventually, that is.